src/SonarScanner.MSBuild.PreProcessor/ArgumentProcessor.cs

```diff
@@ -65,23 +66,23 @@ static ArgumentProcessor()
             Descriptors = new List<ArgumentDescriptor>
             {
                 new ArgumentDescriptor(
-                id: KeywordIds.ProjectKey, prefixes: new string[] { "/key:", "/k:" }, required: true, allowMultiple: false,
+                id: KeywordIds.ProjectKey, prefixes:  CommandLineFlagPrefix.GetPrefixFlag(new string[] { "key:","k:" }), required: true, allowMultiple: false,
```

Comment:
Needs a test that checks that -key, -k, -version, -v etc are actually accepted by the ArgumentProcessor.

Reply:
Ok, added